packages/front-end/components/Marketing/UpgradeMessage.tsx

```diff
@@ -25,8 +27,10 @@ export default function UpgradeMessage({
 
```

Comment:
Once you're at it, do you think you can remove some bad code I left behind on line 26:
`if (process.env.HIDE_GROWTHBOOK_UPGRADE_CTAS) return null;`